twitterscraper/tweet.py

```diff
@@ -74,3 +148,25 @@ def from_html(cls, html):
                     pass  # Incomplete info? Discard!
                 except TypeError:
                     pass  # Incomplete info? Discard!
+
+    @staticmethod
+    def extract_links_and_hashtags(soup_html):
+        links = set()
+        hashtags = set()  # 'hashflag' is hashtag + emoji
+
+        for atag in soup_html.find_all('a'):
+            class_ = atag.get('class', None)
+            if 'hashtag' in atag['href']:
+                # atag['href'] like '/hashtag/NBA?src=hash' -> 'NBA'
+                hashtag = atag['href'].split('?')[0].split('/')[-1]
```

Comment:
You could use a regex here (this is personal preference though). The regular expression could look like this:
`(?<=hashtag/).+(?=\?)`

Also: 
Why not parse the hashtags directly from the tweet text? This could be done with one line of code:
`hashtags = re.findall(r'#\w+', tweet_text)` 
(of course you would need to get the tweet text from somewhere. Maybe pass it as a parameter?)

Reply:
Good recomendation!!
I'll use it.